.gitignore

```diff
@@ -8,7 +8,6 @@ lib/*
 *.log
 *.log.*
 *.csv
-config.json
```

Comment:
I think this should be gitignored, you can force push without changing this!

Reply:
I was thinking about how to do it but I couldn't find a way so I had to do it this way! I'll see what I can do.

Reply:
I just committed the gitignore again back to the original! Now the gitignore change isnt there but the config.json is changed!